ajax/libs/monaco-editor/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "monaco-editor",
+  "version": "0.8.3",
+  "description": "A browser based code editor",
+  "filename": "min/vs/editor/editor.main.js",
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Microsoft/monaco-editor.git"
+  },
+  "homepage": "https://microsoft.github.io/monaco-editor/",
+  "keywords": [
+    "Monaco",
+    "Editor",
+    "TypeScript",
+    "JavaScript"
+  ],
+  "npmName": "monaco-editor",
+  "npmFileMap": [
+    {
+      "basePath": "",
```

Comment:
By the latest line of `min/vs/loader.js` which referred to the map file by relative path, we better keep their structure.